### PR TITLE
fip_77

### PR DESCRIPTION
**Pull request by Joeysantoro**

### CODE REVIEW

File: proposals/description/fip_77.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_77: ProposalDescription = {
+  title: 'FIP-77: Withdraw BentoBox',
+  commands: [
+    {
+      target: 'tribalChief',
+      values: '0',
+      method: 'set(uint256,uint120,address,bool)',
```
Comment (eswak) [inline proposals/description/fip_77.ts:9]: you can use `resetRewards(uint256)` with the poolId, but I guess it doesn't matter the result is the same

File: proposals/dao/fip_77.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  logging && console.log('No Setup');
+};
+
+const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  logging && console.log(`No Teardown`);
+};
+
+const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  logging && console.log('Validate:');
```
Comment (eswak) [inline proposals/dao/fip_77.ts:36]: could add some static checks here on the TribalChief to verify pool allocations

File: proposals/description/fip_77.ts (hunk 1 above)
Comment (Joeysantoro) [reply proposals/description/fip_77.ts:9]: Can't guardian only

File: proposals/dao/fip_77.ts (hunk 2 above)
Comment (Joeysantoro) [reply proposals/dao/fip_77.ts:36]: yes done

**Review by eswak — APPROVED** [review]

**Pull request merged**